martin/src/args/srv.rs

```diff
@@ -10,6 +12,8 @@ pub struct SrvArgs {
     /// Number of web server workers
     #[arg(short = 'W', long)]
     pub workers: Option<usize>,
+    #[arg(help = "to do", short, long)]
+    pub preferred_encoding: Option<String>,
```

Comment:
Don't use String here. Create a new `enum PreferredEncoding` similar to `BoundsCalcType`.  Note that serde supports aliases (i.e. support both br and brotli in one value).